### PR TITLE
Disable safety features of Panda

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: config/franka_control_node.yaml
[the remark this answers is not in the document]
Comment (Kotochleb) [reply config/franka_control_node.yaml:1]: We could load two configs depending on the parameters in the launch file

[the remark this answers is not in the document]
Comment (MedericFourmy) [reply config/franka_control_node.yaml:1]: Sure, let's work on this in this PR before merging I would say

**Review by MaximilienNaveau — APPROVED** [review]

**Comment by MedericFourmy** [comment]
Tested on ciirc panda, it works.

**Review by MedericFourmy — APPROVED** [review]
Tested on ciirc panda, it works as expected.

**Comment by MaximilienNaveau** [comment]
please rebase !

**Comment by MaximilienNaveau** [comment]
Sorry @Kotochleb should fix the conflicts

**Review by MaximilienNaveau — APPROVED** [review]
Please merge

**Pull request merged**